vm/devices/user_driver/src/emulated.rs

```diff
@@ -298,9 +305,13 @@ impl<T: 'static + Send + InspectMut + MmioIntercept> DeviceBacking for EmulatedD
     }
 
     fn map_bar(&mut self, n: u8) -> anyhow::Result<Self::Registers> {
+        if n != 0 {
```

Comment:
For anyone curious, this is both practically fine (we don't map Bar1..6 anywhere in openvmm), and works with the nvme spec. nvme over tcp spec says:

"NVM Express implementations that reside behind PCI compliant bridges, such as PCI Express 
Endpoints, are restricted to having 32-bit assigned base address registers due to limitations on the 
maximum address that may be specified in the bridge. Refer to the PCI Express Base Specification for 
more information on this restriction. " -- https://nvmexpress.org/wp-content/uploads/NVM-Express-PCI-Express-Transport-Specification-Revision-1.1-2024.08.05-Ratified.pdf, section "3.8.1.11 Offset 14h: MUBAR (BAR1) – Memory Register Base Address, upper 32-bits"

Reply:
@gurasinghMS , this may impact some choices you make in `fuzz_nvme_driver` (no need to fuzz anything but Bar0, for example).